src/app/ApplicationTemplate.Shared.Views/Content/Authentication/CreateAccountPage.xaml

```diff
@@ -262,18 +262,18 @@
 					<local:DataValidationView PropertyName="FavoriteDadNames"
 											  Model="{Binding Form}"
 											  Margin="0,4,0,0">
-						<!-- List -->
 						<Grid Background="{StaticResource SurfaceBrush}"
 							  CornerRadius="4">
-							<!--Remove this line from listView Properties to avoid crash on Windows App: ue:ListViewBaseMultipleSelectionBehavior.SelectedItems="{Binding Form.FavoriteDadNames, Mode=TwoWay}"
-								TODO : https://dev.azure.com/nventive/Practice%20committees/_workitems/edit/251910-->
-							<ListView ItemsSource="{Binding DadNames}"
-									  ItemContainerStyle="{StaticResource CheckedListViewItemStyle}"
-									  SelectionMode="Single"
-									  IsItemClickEnabled="True" />
+							<ItemsControl ItemsSource="{Binding DadNames}">
```

Comment:
`ListView` and `ItemsControl` are not the same. `ItemsControl` doesn't have any selection support. We need to be able to use `ListView` in this kind of context.

Reply:
if we use Listview then voiceover doesn't work nor the toggle function when voiceover is enabled :/... should i look for a workaround using this ItemsControl??

Reply:
It must work with `ListView`. It sounds like Uno issues.

Reply:
I would suggest to use a ListView and add a comment on top "Voiceover doesn't work see LINK for more details" with the link pointing to your Uno issue

Reply:
Uno issue created
https://github.com/unoplatform/uno/issues/12329